samples/suit/smp_transfer/sysbuild/hci_ipc_extflash.conf

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+CONFIG_BT_BUF_ACL_TX_SIZE=502
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+# Disable logs over UART
+CONFIG_LOG=n
+CONFIG_LOG_PRINTK=n
+CONFIG_UART_CONSOLE=n
+
+# Do not generate or merge the MPI generated by the hci_ipc image.
+# It would result in the MPI configuration managed by the application core (main)
+# image to be overwritten by the default configuration from hci_ipc.
+CONFIG_SUIT_MPI_GENERATE=n
+CONFIG_SUIT_ENVELOPE_OUTPUT_MPI_MERGE=n
```

Comment:
Please align with: https://github.com/nrfconnect/sdk-nrf/pull/16815/files

Reply:
Not sure if I understand, the config is mostly a copy of `smp_transfer/sysbuild/hci_ipc.conf`. This config affects the `hci_ipc` that is used by the `smp_transfer` application, not the recovery application. Therefore it needs the UICRs to be generated.

Reply:
Ah, I thought this was for the flash companion, my mistake.